api/client/cp.go

```diff
@@ -21,17 +21,16 @@ func (cli *DockerCli) CmdCp(args ...string) error {
 
 	cmd.ParseFlags(args, true)
 
-	var copyData engine.Env
 	info := strings.Split(cmd.Arg(0), ":")
 
 	if len(info) != 2 {
 		return fmt.Errorf("Error: Path not specified")
 	}
 
-	copyData.Set("Resource", info[1])
-	copyData.Set("HostPath", cmd.Arg(1))
-
-	stream, statusCode, err := cli.call("POST", "/containers/"+info[0]+"/copy", copyData, nil)
+	cfg := &types.CopyConfig{
+		Resource: info[1],
```

Comment:
So glad to see we're not passing HostPath to the server any more. I had a PR for that but it was rejected.